plugins/extractors/bigquery/bigquery.go

```diff
@@ -477,6 +483,13 @@ func (e *Extractor) getColumnMode(col *bigquery.FieldSchema) string {
 	}
 }
 
+func (e *Extractor) getPolicyTagList(col *bigquery.FieldSchema) []string {
```

Comment:
This is unnecessary. `len` will work fine with a `nil` slice as well and return 0 as expected.

Reply:
Apologies, looks like this review comment was incorrect.